src/3rd_party_repos.c

```diff
@@ -263,12 +263,19 @@ enum swupd_code third_party_set_repo(const char *state_dir, const char *path_pre
 	char *repo_path_prefix;
 	char *repo_cert_path;
 
+	/* TODO(castulo): we may want to inform the user that some global
```

Comment:
is this still valid?

Reply:
no, per the discussion we had over IRC the other day we no longer need this TODO... I'll remove it.